test/model/emoji_test.dart

```diff
@@ -246,15 +248,16 @@ void main() {
     });
 
     test('updates on setServerEmojiData', () {
-      final store = prepare();
+      final store = prepare(unicodeEmoji: null, addServerDataForPopular: false);
+      check(store.debugServerEmojiData).isNull();
```

Comment:
Ah, good thought checking this is null (following up on https://github.com/zulip/zulip-flutter/pull/1506#discussion_r2093758975 ).